lib/rules/selector-no-qualifying-type/index.mjs

```diff
@@ -13,7 +13,9 @@ import validateOptions from '../../utils/validateOptions.mjs';
 const ruleName = 'selector-no-qualifying-type';
 
 const messages = ruleMessages(ruleName, {
-	rejected: (selector) => `Unexpected qualifying type selector "${selector}"`,
+	rejected: (tag, selector) => {
```

Comment:
[suggestion] If we'd like to keep compatibility of the functional `message` secondary option, it might be better to add a new `tag` at last and keep the existing `selector` argument position:

```suggestion
	rejected: (selector, tag) => {
```

Ref:
https://github.com/stylelint/stylelint/blob/afe57f9a524aa5a8106521ae65543ec47b7f033e/docs/user-guide/configure.md?plain=1#L154-L156

Reply:
Good thinking! Thanks